.github/workflows/labels.yml

```diff
@@ -12,7 +12,7 @@ jobs:
     steps:
     - uses: actions/setup-node@v2
       with:
-        node-version: '14'
+        node-version: '16'
```

Comment:
This doesn't need to be bumped though.

Reply:
Node 16 is the new LTS

Reply:
But do we need to bump the action? It works fine in v14 😅 but yeah, we could switch it.

Reply:
This is the labeller action, which isn't directly related to the code, but it's still good to have consistency in the environment settings IMHO.